src/components/Footer/index.jsx

```diff
@@ -0,0 +1,18 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHeart } from '@fortawesome/free-solid-svg-icons';
+import './styles.css';
+
+export const Footer = ({ phrase = 'Designed with' }) => {
+  return (
+    <>
+      <footer className="footer">
+        <p>
+          {phrase}
+          <span>
+            <FontAwesomeIcon icon={faHeart} />
```

Comment:
I think icon could be passed as a prop, because on this way the footer always will haves a hear

if you have `phrase = "Bazinga!!"`

the footer will shows

Bazinga!! ❤️